### PR TITLE
 Delfin Metric List

### PR DESCRIPTION
**Pull request by NajmudheenCT**
**What type of PR is this?**

> /kind design

**What this PR does / why we need it**:
 List all first set delfin unified metric names, unit and description based on hetrogenious devices and SRM platforms  metric list analysis
**Which issue(s) this PR fixes**:

Fixes # https://github.com/sodafoundation/delfin/issues/570

**Test Report Added?**:
NA

**Test Report**:

**Special notes for your reviewer**:

### CODE REVIEW

File: arch-design/delfin/Delfin_Metric_List.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| Metric Name     | Unit | Description                             |
+| --------------- | ---- | --------------------------------------- |
+| iops            | iops | read and write  operations per second   |
+| readIops        | iops | Read  operations per second             |
+| writeIops       | iops | Write  operations per second            |
+| throughput      | MB/S | Total data transferred per second       |
+| readThroughput  | MB/S | Total read data transferred per second  |
+| writeThroughput | MB/S | Total write data transferred per second |
+| responseTime    | ms   | Average time taken for an IO operation  |
+### Port
```
Comment (kumarashit) [inline arch-design/delfin/Delfin_Metric_List.md:63]: nit: Just put a new line before ### Port

File: arch-design/delfin/Delfin_Metric_List.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+| Metric Name     | Unit | Description                             |
+| --------------- | ---- | --------------------------------------- |
+| iops            | iops | read and write  operations per second   |
+| readIops        | iops | Read  operations per second             |
+| writeIops       | iops | Write  operations per second            |
+| throughput      | MB/S | Total data transferred per second       |
+| readThroughput  | MB/S | Total read data transferred per second  |
+| writeThroughput | MB/S | Total write data transferred per second |
+| responseTime    | ms   | Average time taken for an IO operation  |
+### Disk
```
Comment (kumarashit) [inline arch-design/delfin/Delfin_Metric_List.md:74]: Same here..Please check all

File: arch-design/delfin/Delfin_Metric_List.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+| Metric Name        | Unit | Description                                 |
+| ------------------ | ---- | ------------------------------------------- |
+| iops               | iops | read and write  operations per second       |
+| readIops           | iops | Read  operations per second                 |
+| writeIops          | iops | Write  operations per second                |
+| throughput         | MB/S | Total data transferred per second           |
+| readThroughput     | MB/S | Total read data transferred per second      |
+| writeThroughput    | MB/S | Total write data transferred per second     |
+| responseTime       | ms   | Average time taken for an IO operation      |
+| cacheHitRatio      | %    | Percentage of io that are cache hits        |
```
Comment (kumarashit) [inline arch-design/delfin/Delfin_Metric_List.md:45]: nit: Change io -> I/O or IO -> I/O everywhere

**Review by kumarashit — APPROVED** [review]
LGTM

**Review by skdwriting — APPROVED** [review]
LGTM

**Pull request merged**